etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/messages/SendMessageHelper.java

```diff
@@ -20,7 +20,14 @@ public static SendMessageHelper getInstance() {
     private SendMessageHelper() {}
 
     public void savePartnerMetadataForReceivedMessage(
-            String receivedSubmissionId, int messageHash, PartnerMetadataMessageType messageType) {
+            String receivedSubmissionId,
```

Comment:
Thoughts on passing an object that will hold all this data so we can have one parameter?

Reply:
I'm down for something like `PartnerMetadataMessage` or something along those lines. I would like others thoughts on it.

Reply:
Hmmmm, I'm fine passing along the individual arguments, but I wouldn't create a whole new class when a lot of this data just belongs in `PartnerMetadata`.  Today we construct the `PartnerMetadata` _inside_ `PartnerMetadataOrchestrator`.  I'd be curious to hear people's thoughts on constructing `PartnerMetadata` in the usecase or somewhere else outside the orchestrator.

Reply:
Having an object instead of many params will be slightly more readable at this point. I don't have super strong feelings on it though

Reply:
Maybe we can bubble up the construction of the `PartnerMetadata`? That could lead to easier change to that object in the future

Reply:
I'll leave this as is for now and we can discuss this in our engineering block if we want to consider refactoring.